### PR TITLE
Added command to clone the repository

### PR DESCRIPTION
**Pull request by Abhinavcode13**
@jafingerhut , PTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,12 +99,16 @@ virtual machine or install several dependencies.
 
 1. Install Vagrant and VirtualBox on your system.
 2. Clone the repository
-3. Navigate to the cloned directory :
+   
+   ```
+   https://github.com/p4lang/tutorials.git
+   ```
+4. Navigate to the cloned directory :
```
Comment (jafingerhut) [inline README.md:106]: Why are you omitting step number 3?

Comment (jafingerhut) [reply README.md:106]: To clarify, the changes as I am reviewing them now change a list of steps numbered 1,2,3,4 to a list of steps numbered 1,2,4,5, which does not look correct to me.

File: README.md [hunk 2]
```diff
@@ -99,12 +99,16 @@ virtual machine or install several dependencies.
 
 1. Install Vagrant and VirtualBox on your system.
 2. Clone the repository
-3. Navigate to the cloned directory :
+   
+   ```
+   https://github.com/p4lang/tutorials.git
```
[the remark this answers is not in the document]
Comment (Abhinavcode13) [reply README.md]: Updated

**Review by jafingerhut — APPROVED** [review]
LGTM

**Pull request merged**